base/refpointer.jl

```diff
@@ -137,6 +137,8 @@ if is_primary_base_module
 
     Ref(x::Ref, i::Integer) = (i != 1 && error("Ref only has one element"); x)
     Ref(x::Ptr{T}, i::Integer) where {T} = x + (i - 1) * Core.sizeof(T)
+    #For multidimensional-arrays
+    Ref(x::AbstractArray{T},row::Integer,column::Integer) where {T} = Ref(x,row+(column-1)*size(x)[1])
```

Comment:
This should also not have been changed — did you accidentally pull in an unrelated change?

Reply:
Sir, I was working on a different issue ( #39672 ). This by mistake got pulled in this branch. I have removed this code.